openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java

```diff
@@ -341,6 +345,16 @@ private Patient preparePatient() {
         return new Patient(person, MOTECH_ID, location);
     }
 
+    private void updatePatient() {
+        Attribute.AttributeType attributeType = new Attribute.AttributeType();
+        attributeType.setUuid("8d8718c2-c2cc-11de-8d13-0010c6dffd0f");
+        Attribute personAttribute = new Attribute();
+        personAttribute.setAttributeType(attributeType);
+        personAttribute.setValue("New York");
```

Comment:
additionally you can add few new lines into this method, it will be more readable